.gitmodules

```diff
@@ -152,3 +146,6 @@
 [submodule "projects/mcgill-emc-rna-seq-experiment"]
 	path = projects/mcgill-emc-rna-seq-experiment
 	url = https://github.com/conpdatasets/mcgill-emc-rna-seq-experiment.git
+[submodule "projects/preventad-open"]
+	path = projects/preventad-open
+	url = https://github.com/conpdatasets/preventad-open/
```

Comment:
This is the result of using `git submodule add <submodule>` to put the submodules back; I can manually edit to add `.git` to the end of each line if appropriate.

Reply:
I think it should have the .git at the end like all other datasets. Same for the openpain below.

I guess the `submodule add` will work regardless if you give the URL with the .git or not.  It would mainly be to stay consistant with the other submodules.